pages/Using Rollbax in Plug-based applications.md

```diff
@@ -50,4 +50,20 @@ defp handle_errors(conn, %{kind: kind, reason: reason, stack: stacktrace}) do
 end
 ```
 
+#### Sensitive Data
```

Comment:
Can we move this section at the very end, after the "Check the documentation for the Rollbar API [...]" part?